architecture/additional_concepts/authorization.adoc

```diff
@@ -399,10 +405,16 @@ Admission uses the following approach to create the final security context for
 the pod:
 
 . Retrieve all SCCs available for use.
-. Generate field values for any security context setting that was not specified
+. Generate field values for security context settings that were not specified
 on the request.
 . Validate the final settings against the available constraints.
 
+For a *FSGroup* SCC strategy of `MustRunAs`, if the pod spec defines a _fsGroup_ id then that id value must equal the default fSGroup id (first id in the first range in the namespace's "openshift.io/sa.scc.supplemental-groups" annotation). Otherwise, the pod is not authorized by that SCC and the next SCC is evaluated.
+If the *FSGroup* setting is `RunAsAny`, or if the pod spec omits a _fsGroup_ id then the pod matches the SCC based on FSGroup.
+
+For a *SupplementalGroups* SCC strategy of `MustRunAs`, if the pod spec defines a _SupplementalGroups_ id(s) then the pod's id(s) must equal one of the ids in the namespace's "openshift.io/sa.scc.supplemental-groups" annotation. Otherwise, the pod is not authorized by that SCC and the next SCC is evaluated.
```

Comment:
So, basically, an admin can use the api to add any security context they desire, eg. the various settings for v1.SecurityContextConstraints, defined here: https://docs.openshift.org/latest/rest_api/kubernetes_v1.html#v1-securitycontextconstraints
My experience has been using only the predefined SCCs, so thanks for the more global view!